client/src/components/ResultPanel.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Uri, ViewColumn, WebviewPanel, l10n, window } from "vscode";
+
+import {
+  isSideResultEnabled,
+  isSinglePanelEnabled,
+} from "./Helper/SettingHelper";
+
+let resultPanel: WebviewPanel | undefined;
+
+export const showResult = (html: string, uri?: Uri, title?: string) => {
+  const sideResult = isSideResultEnabled();
+  const singlePanel = isSinglePanelEnabled();
+
+  if (!singlePanel || !resultPanel) {
```

Comment:
Am I understanding this code, right? Basically, it's saying:
 - If there is not a panel, create or
 - else, update the current panel title and contents? 

If that's the case, it might be helpful to have something like `const panelExists = !singlePanel || !resultPanel` (my brain was having a bit of trouble understanding it)

Reply:
Well, there're 2 conditions.
`!resultPanel` means there's not a panel.
`!singlePanel` means user don't want to reuse. So we'll create no matter panel exists.
`resultPanel` is the panel instance. `singlePanel` is the setting value. Does it make sense?